### ISSUE TITLE
Larger tap targets for separators

### ISSUE THREAD

**Issue by hacker1024** [ibako/resizable_widget#16]
Sometimes, it looks nicer to have a more `Divider`-like separator with a very thin thickness.
If the thickness is set to a small number, however, the separator becomes very hard to drag.

It'd be great if there was a way to increase the tap target of the separator without making it visually thicker.

**Comment by bidica06** [comment]
I agree! Sometimes it is hard to drag it on touch screens devices creating a bad user experience.
